packages/block-library/src/post-featured-image/index.php

```diff
@@ -35,15 +35,15 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	}
 
 	if ( $has_width ) {
-		$wrapper_attributes = get_block_wrapper_attributes( array( 'style' => "width:{$attributes['width']};" ) );
+		$wrapper_attributes = get_block_wrapper_attributes( array( 'style' => esc_attr( "width:{$attributes['width']};" ) ) );
 	}
 
 	if ( $has_height ) {
 		$image_styles = "height:{$attributes['height']};";
 		if ( ! empty( $attributes['scale'] ) ) {
 			$image_styles .= "object-fit:{$attributes['scale']};";
 		}
-		$featured_image = str_replace( 'src=', "style='$image_styles' src=", $featured_image );
+		$featured_image = str_replace( 'src=', 'style="' . esc_attr( $image_styles ) . '" src=', $featured_image );
```

Comment:
Is there a reason why the `"` was changed to `'` in `src`? 🤔

Reply:
I am getting WordPress Coding Standards warnings about using double quotes for strings. That said I know both are deemed acceptable. 

But as we're not interpolating variables anymore single quotes are probably best suited here.